### ISSUE TITLE
seqtk/rename conda test fails

### ISSUE THREAD

**Issue by Emiller88** [nf-core/modules#1417]
Follow up to #1405 
Probably the same thing as #1416 

```
=================================== FAILURES ===================================
_________________________________ test session _________________________________
Observed md5sum '05dd5b3714358ec75d78f72b2729c963' not equal to expected md5sum 'babdfc2a3940a1e32a63479db2c1d600' for file '/home/runner/pytest_workflow_r6zhm_9y/seqtk_rename_test_seqtk_rename_fq/output/seqtk/test.renamed.fastq.gz'
```

**Comment by sateeshperi** [comment]
@Emiller88 any tips on how to fix this?

**Comment by Emiller88** [comment]
@sateeshperi, my bad, I should have included from #1416 
> Only the conda tests fail, singularity and docker both pass so I'm guessing it's an error with the package version in conda.
So I'd check that the software versions match up in the containers with what's specified in the conda env.

**Comment by muffato** [comment]
Fixed by #2076
